libs/SalesforceHybridSDK/SalesforceHybridSDK.xcodeproj/project.pbxproj

```diff
@@ -1506,6 +1506,7 @@
 			baseConfigurationReference = CE8F13091AD4A87200233F2F /* SalesforceHybridSDK-Test.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(BUILT_PRODUCTS_DIR)/SalesforceHybridSDKTestApp.app/SalesforceHybridSDKTestApp";
+				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
```

Comment:
(I put a line about it in my PR description but it got eaten up by Markdown)

We are getting the error (non-modular includes) for all the &#35;import &lt;Cordova/xxx&gt;